helpers/web.dev.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "web.dev",
+  "desc": "Get the web's modern capabilities on your own sites and apps with useful guidance and analysis from web.dev",
+  "url": "https://web.dev/measure/",
+  "tags": [
+    "Site analyzers"
+  ],
+  "maintainers": [
+    "googlechrome"
```

Comment:
```suggestion
    
```

So far we're only listing maintainers when it's on GitHub and maintained by a person. :)